src/views/Community/View.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+// import Dashboard from '../Dashboard'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { Row, Progress } from 'reactstrap';
+import { sendCommunityData } from '../../store/actions/community'
+import Payment from '../../components/Payment'
+import { isAuthUserType } from '../../services/Auth'
+import { Link } from 'react-router-dom'
+import Page404 from '../../views/Pages/Page404'
+
+
+class View extends Component {
+  constructor(props) {
+    super(props)
+
+    this.deleting = this.deleting.bind(this)
+  }
+
+  componentDidMount() {
+    let id = this.props.match.params.id * 1;
+    let { sendData, communityData } = this.props
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['sendData' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)